demo/src/Generator.tsx

```diff
@@ -1,4 +1,5 @@
-import { HierarchicalNodeProps, TreeDataType } from 'react-wooden-tree';
+// import { HierarchicalNodeProps, TreeDataType } from 'react-wooden-tree';
+import { HierarchicalNodeProps, TreeDataType } from '../../src/';
```

Comment:
This was used with `npm link` to show that it is imported from the package, but I guess, it is more convenient like this.

Reply:
ah, looks like we can keep the name and use an alias, that also works :)